project-codeflare/codeflare-operator
# Copying MCAD CRDs in Kustomize config instead of referencing them

# Issue link
Closes #353 

# What changes have been made
Refactored to copy MCAD CRDs instead of referencing them.

# Verification steps
- Run `make manifests` and ensure that the config/crd/mcad/kustomization.yaml shows the mcad version (not 0.0.0.0).
- Install MCAD in a cluster by running `make install` and ensure that the required CRDs have been created and that everything runs as expected.

## Checks
- Testing Strategy
